tutorials/promscale-how-it-works.md

```diff
@@ -0,0 +1,164 @@
+* SQL queries are handled by TimescaleDB directly.
+
+As can be seen, this architecture has relatively few components, enabling simple operations.
+
+#### Promscale PostgreSQL Extension
+
+Promscale has a dependency on the [Promscale PostgreSQL extension][promscale-extension], which contains support functions to improve the performance of Promscale. While Promscale is able to run without the additional extension installed, adding this extension will get better performance from Promscale. 
+
+#### Deploying Promscale
+
+Promscale can be deployed in any environment running Prometheus, alongside any Prometheus instance. We provide Helm charts for easier deployments to Kubernetes environments (see Section 3 of this tutorial for more on installation and deployment).
+
+### 2.2 Promscale Schema [](promscale-schema)
+
+To achieve high ingestion, query performance and optimal storage we have designed a schema which takes care of writing the data in the most optimal format for storage and querying in TimescaleDB. Promscale translates data from the [Prometheus data model][prometheus-native-format] into a relational schema that is optimized for TimescaleDB, is stored efficiently, and is easy to query.
+
+The basic schema uses a normalized design where the time-series data is stored in compressed hypertables. These tables have a foreign-key to series tables (stored as vanilla PostgreSQL tables), where each series consists of a unique set of labels.
+
+In particular, this schema decouples individual metrics, allowing for the collection of metrics with vastly different cardinalities and retention periods. At the same time, Promscale exposes simple, user-friendly views so that you do not have to understand this optimized schema (see 2.3 for more on views).
+
+> :TIP: Promscale automatically creates and manages database tables. So, while understanding the schema can be beneficial (and interesting), it is not required to use Promscale. Skip to Section 2.3 for information how to interact with Promscale using SQL views and to Section 4 to learn using hands on examples.
+
+
+#### 2.2.1 Metrics Storage Schema
+Each metric is stored in a separate hypertable. 
+
+A hypertable is a TimescaleDB abstraction that represents a single logical SQL table that is automatically physically partitioned into chunks, which are physical tables that are stored in different files in the filesystem. Hypertables are partitioned into chunks by the value of certain columns. In this case, we will partition out tables by time (with a default chunk size of 8 hours).
+
+**Compression**
+
+The first chunk will be decompressed to serve as a high-speed query cache. Older chunks are stored as compressed chunks. We configure compression with the segment_by column set to the series_id and the orderby column set to time DESC. These settings control how data is split into blocks of compressed data. Each block can be accessed and decompressed independently.
```

Comment:
> The *first* chunk will be decompressed...

"latest"?

Reply:
"segment_by", "orderby" are used here without any prior reference?

also, should it be "order_by"?

Reply:
Fixed